### ISSUE TITLE
Host Crash Loop + Error Reporting

### ISSUE THREAD

**Issue by mathewc** [Azure/azure-webjobs-sdk-script#1644]
It appears that in some cases when the runtime is in a crash loop (e.g. due to an incorrectly configured ServiceBus trigger connection string) the portal will sometimes stop showing the actual underlying error and will instead show a generic 500 error with no details.

Repro steps:

- Create a ServiceBus queue triggered function for a queue that doesn't exist
- set the connection to use a SB connection that doesn't have Manage rights

The host will enter an error loop in this case, because it is trying to create the queue but doesn't have rights. In the portal, initially you'll see the correct error message, but if you dismiss it and wait for a couple minutes, you'll start getting the bad error message (just generic 500).

**Comment by christopheranderson** [comment]
TL;DR - When you're in a crash loop, we need to provide better feedback that there is something permanently broken about your host.

**Comment by paulbatum** [comment]
@mathewc Why does a bad service bus connection string result in a crash loop? I thought indexing for that function would fail and we would still bring the host up?
